code/game/objects/items.dm

```diff
@@ -105,6 +105,8 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 	var/datum/rpg_loot/rpg_loot = null
 
+	var/held = null

```

Comment:
Well null is wrong in all of them and would behave the exact same as ```FALSE``` in terms of being spawned to hands.
and also, spawning shit into people's hands DOES call the appropriate inventory procs, so, ```FALSE``` please.